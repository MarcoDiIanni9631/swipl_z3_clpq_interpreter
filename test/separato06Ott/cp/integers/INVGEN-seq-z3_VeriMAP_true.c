# 1 "MAP/SAFE-exbench/INVGEN-seq-z3.tmp.c"
# 1 "<command-line>"
# 1 "MAP/SAFE-exbench/INVGEN-seq-z3.tmp.c"
# 21 "MAP/SAFE-exbench/INVGEN-seq-z3.tmp.c"
void main() {
  int n0, n1;
  int i = 0;
  int k = 0;

  while( i < 20*n0 ) {
    i++;
    k++;
  }
  i = 0;
  while( i < 6*n1+128 ) {
    i++;
    k++;
  }
  i = 0;
  while( i < 6*n1+128 ) {
    i++;
    k--;
  }
  i = 0;
  while( i < 20*n0 ) {
    __VERIFIER_assert( k > 0 );
    i++;
    k--;
  }
  return 0;
}
