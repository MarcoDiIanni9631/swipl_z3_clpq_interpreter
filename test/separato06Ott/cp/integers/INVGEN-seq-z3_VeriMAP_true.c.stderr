MAP/SAFE-exbench/INVGEN-seq-z3.tmp.c:46: Warning: Return statement with a value in function returning void
